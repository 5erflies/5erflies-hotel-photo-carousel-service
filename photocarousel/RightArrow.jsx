import React, { useState, useEffect } from 'react';
import styled from 'styled-components';

const RightArrow = ({addCount}) => (
  // error: photoCount is read only??
  <div>
    <Arrow onClick={addCount}>
      >
    </Arrow>
  </div>

  // const LeftArrow = () => (
  //   <Arrow onClick={MinusCount}>
  //     Hi
  //   </Arrow>
  // )
)

const Arrow = styled.button`
  border-radius: 50%
  border: 1px solid  #C5C5C5;
  font-size: 16px;
  background-color: white;
`;

  export default RightArrow;