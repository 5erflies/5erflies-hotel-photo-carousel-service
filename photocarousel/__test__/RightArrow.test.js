import React from 'react';
import styled from 'styled-components';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import RightArrow from '../RightArrow.jsx';

const addCount = () => setphotoCount(photoCount++);

describe('<Header> components', () => {
  it('should match snapshot created', () => {
    const tree = renderer.create(<RightArrow addCount={addCount} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});