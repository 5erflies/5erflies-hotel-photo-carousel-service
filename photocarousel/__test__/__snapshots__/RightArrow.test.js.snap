// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header> components should match snapshot created 1`] = `
<div>
  <button
    className="sc-bdVaJa glYYXl"
    onClick={[Function]}
  >
    &gt;
  </button>
</div>
`;
